zerver/models.py

```diff
@@ -152,6 +152,7 @@ class Realm(ModelReprMixin, models.Model):
     date_created = models.DateTimeField(default=timezone.now) # type: datetime.datetime
     notifications_stream = models.ForeignKey('Stream', related_name='+', null=True, blank=True) # type: Optional[Stream]
     deactivated = models.BooleanField(default=False) # type: bool
+    default_language = models.CharField(default=u'en', max_length=50) # type: text_type
```

Comment:
I guess actually a nice thing you could do in this branch is a quick refactoring commit rebased to the start of your branch that moves the value 50 to be a constant at the top of `zerver/models.py` with a nice name like `MAX_LANGUAGE_ID_LENGTH`, and we can merge that commit immediately; then the ticket can be just about changing this value.